### ISSUE TITLE
Address bar clears on page visit. Also doesn't refresh page on re-enter.

### ISSUE THREAD

**Issue by deanpress** [nos/client#65]
2 bugs:
- [x]  The address bar clears on visit.
- [x] Entering the same URL of the visited page doesn't "refresh" the page.

![](https://i.gyazo.com/b1efc1c6b18faf6b826c32428fa5cbf6.gif)

**Comment by kitchenSpoon** [comment]
Would removing `if (value !== query) ` check  from
```
handleKeyUp = (event) => {
    const { query, doQuery } = this.props;

    if (event.which === RETURN_KEY) {
      const { value } = event.target;

      if (value !== query) {
        doQuery(value);
      }
      this.props.history.push('/browser');
    }
  };
```

in AddressBar.js do the trick or is it something more sophisticated?

**Comment by mhuggins** [comment]
This would mostly be fixed by #117, but I'm running into issues with getting it merged because my change is causing the webview to crash, and I haven't figured out why yet.  Hopefully I can resolve that and merge.  I'll also be working on browser tabs this week, which will touch on this issue.

**Comment by mhuggins** [comment]
This should now be working as expected.
